tasks/db_deploy/db_deploy.py

```diff
@@ -60,11 +62,14 @@ def task(config: Dict[str, str]) -> None:
 
     # Connect as admin user to the postgres database
     with postgres_admin_engine.connect() as connection:
-        # Check if database exists, if not throw an error
+        # Check if database exists. If not, start from scratch.
         if not app_db_exists(connection):
             logger.critical("The ORCA database disaster_recovery does not exist.")
-            # TO DO ORCA-233: Add db creation code here and remove the raise Exception error
-            raise Exception("Missing application database.")
+            connection.execute(orca_sql.commit_sql())  # exit the default transaction to allow database creation.
```

Comment:
Instead of a commit_sql why not just run the single "commit" command here? Similar to what is done in the other connection blocks.

Reply:
I can do that once, but beyond that we get into the problem with test_migrate_db where we aren't checking what order the text( calls are used in, just what order they're constructed.

Reply:
Sounds good